src/ui/MultipleFilesMessageItemBody/index.tsx

```diff
@@ -49,7 +56,7 @@ export default function MultipleFilesMessageItemBody({
     );
   }
 
-  return (
+  return imageSideLength && (
```

Comment:
This might cause print `0` on the screen unexpectedly. Here's why 
https://react.dev/learn/conditional-rendering#logical-and-operator-
<img width="1000" alt="Screenshot 2023-09-04 at 6 09 30 PM" src="https://github.com/sendbird/sendbird-uikit-react/assets/10060731/6cc85176-1077-4995-9c8b-cb8e0b03fe7b">


```suggestion
  return imageSideLength > 0 && (
```

It;s better to be more specific here especially when the condition is `xxx.length` (which can be 0).

Reply:
Oh normally `length` indicates number than string so I thought the value would be 0 than ''.

Reply:
yes its true but I chose this name over 'xxxInPx' bc then i have to change name of reducer etc. I think given that width, height is in pixels, side length can also be perceived the same.